src/Toast/Toast.test.tsx

```diff
@@ -55,38 +53,50 @@ describe('<Toast />', () => {
   });
   it('autohide is set to false on onMouseOver and true on onMouseLeave', async () => {
     render(
-      <ToastWrapper data-testid="toast" {...props}>
+      <ToastWrapper {...props}>
         Success message.
       </ToastWrapper>,
     );
-    const toast = screen.getByTestId('toast');
+    const toast = screen.getByRole('alert');
     await userEvent.hover(toast);
     setTimeout(() => {
-      expect(screen.getByText('Success message.')).toEqual(true);
+      expect(screen.getByText('Success message.')).toBeTruthy();
```

Comment:
[observation] no change needed, just an callout we may want to generally opt for more explicit assertions than `.toBeTruthy`  moving forward (e.g., `.toBeInTheDocument()`, `.toBeVisible()`, etc.).

Reply:
Good call! It might make sense to document testing best practices like this in an OEP - possibly a good topic for a Frontend Working Group meeting.